src/backgroundblurprocessor/BackgroundBlurStrength.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const BlurStrength = {
+  LOW: 7,
```

Comment:
Would be great to have any explanation on the numbers, we have got issues when it came to active speaker policy scoring mechanism for which we did not had enough documentation on the numbers, hence would appreciate if we can add documentation here if any.

Reply:
added a comment